internal/logic_handlers/organizationlogic/service_test.go

```diff
@@ -0,0 +1,680 @@
+
+				return &dependencies{
+					Persistor:  &organizationlogicfakes.FakePersistor{},
+					TxProvider: &mockTxProvider,
+					Logger:     mockLogger,
+					Cleanup:    cleanup,
+				}, cleanup
+			},
+			args: argsGetOrganizations{
+				ctx:    context.Background(),
+				filter: &model.OrganizationFilters{},
+			},
+			mutations: func(t *testing.T, db *sqlx.DB) {},
+			assertions: func(t *testing.T, paginated *model.PaginatedOrganizations, err error) {
+				require.Error(t, err, "unexpected get organizations error")
+				require.Contains(t, err.Error(), "get db:")
+			},
+		},
+	}
+}
+
+func TestService_GetOrganizations(t *testing.T) {
+	for _, tt := range getGetOrganizationTestCases() {
+		t.Run(tt.name, func(t *testing.T) {
+			_dependencies, cleanup := tt.getDependencies(t)
+			defer cleanup()
+
+			svc, err := New(&Config{
+				TxProvider: _dependencies.TxProvider,
+				Logger:     _dependencies.Logger,
+				Persistor:  _dependencies.Persistor,
+			})
+			require.NoError(t, err, "unexpected new error")
+
+			tt.mutations(t, _dependencies.Db)
+
+			paginatedOrganizations, err := svc.ListOrganizations(tt.args.ctx, tt.args.filter)
+			tt.assertions(t, paginatedOrganizations, err)
+		})
+	}
+}
+
+type testCaseUpdateOrganization struct {
+	name            string
+	getDependencies func(t *testing.T) (*dependencies, func(ignoreErrors ...bool))
+	ctx             context.Context
+	mutations       func(t *testing.T, db *sqlx.DB) (toUpdatedOrganization *model.UpdateOrganization)
+	assertions      func(t *testing.T, params *model.UpdateOrganization, organization *model.Organization, err error)
+}
+
+func getTestCasesUpdateOrganizations() []testCaseUpdateOrganization {
+	return []testCaseUpdateOrganization{
+		{
+			name:            "success",
+			ctx:             context.TODO(),
+			getDependencies: getConcreteDependencies,
+			mutations: func(t *testing.T, db *sqlx.DB) (toUpdatedOrganization *model.UpdateOrganization) {
+				user, err := mysqlmodel.Users().One(context.TODO(), db)
+				require.NoError(t, err, "error finding user from user table.")
+
+				entry := mysqlmodel.Organization{
+					ID:            user.ID,
+					Name:          "Demby",
+					CreatedBy:     null.IntFrom(user.ID),
+					LastUpdatedBy: null.IntFrom(user.ID),
+					CreatedAt:     time.Now(),
+					LastUpdatedAt: null.TimeFrom(time.Now()),
+					IsActive:      true,
+				}
+
+				err = entry.Insert(context.Background(), db, boil.Infer())
+				require.NoError(t, err, "error inserting sample data")
+
+				toUpdateOrganization := &model.UpdateOrganization{
+					Id: user.ID,
+					Name: null.String{
+						String: "Lawrence",
+						Valid:  true,
+					},
+					UserId: null.Int{
+						Int:   user.ID,
+						Valid: true,
+					},
+				}
+
+				return toUpdateOrganization
+			},
+			assertions: func(t *testing.T, params *model.UpdateOrganization, organization *model.Organization, err error) {
+				createdByConvToInt, _ := strconv.Atoi(organization.CreatedBy)
```

Comment:
add error handling:
createdByConvToInt, err := strconv.Atoi(organization.CreatedBy)
require.NoError(t, err, "unexpected strconvErr") // non-verbatim don't just copy and paste, but check syntax

Reply:
Error handling is now added.  Test passing.